deploy/sample-backup-configs/sample_config_build_recipes.ini

```diff
@@ -36,14 +36,14 @@ deploytriplet=None
 [firesim-quadcore-nic-ddr3-llc4mb]
 DESIGN=FireSim
 TARGET_CONFIG=FireSimRocketChipQuadCoreConfig
-PLATFORM_CONFIG=FireSimDDR3FRFCFSLLC4MBConfig90MHz
+PLATFORM_CONFIG=F90MHz_FireSimDDR3FRFCFSLLC4MBConfig
 instancetype=c4.4xlarge
 deploytriplet=None
 
 [firesim-quadcore-no-nic-ddr3-llc4mb]
 DESIGN=FireSimNoNIC
 TARGET_CONFIG=FireSimRocketChipQuadCoreConfig
-PLATFORM_CONFIG=FireSimDDR3FRFCFSLLC4MBConfig90MHz
+PLATFORM_CONFIG=F90MHz_ireSimDDR3FRFCFSLLC4MBConfig
```

Comment:
the second part of this is broken

Reply:
Will fix if we're down to make this change.